bftengine/src/bftengine/ReplicaImp.cpp

```diff
@@ -2208,7 +2202,8 @@ void ReplicaImp::onFastPathCommitCombinedSigSucceeded(SeqNum seqNumber,
   auto span = concordUtils::startChildSpanFromContext(fcp->spanContext<std::remove_pointer<decltype(fcp)>::type>(),
                                                       "bft_execute_committed_reqs");
 
-  metric_total_committed_sn_++;
+  auto commitPath = seqNumInfo.getFastPathSelfPartialCommitProofMsg()->commitPath();
```

Comment:
The commit path is already known as a function argument of onFastPathCommitCombinedSigSucceeded.

Reply:
Changed to use the passed parameter